services/data/data/dependencies/settings.py

```diff
@@ -140,19 +140,20 @@ def initialise_settings() -> None:
     settings.validators.register(
         Validator(SettingKeys.CACHE_PATH.value, must_exist=True, condition=utils.check_create_folder, when=not_doc),
 
-        # Validator("CSW_SERVER", must_exist=True, when=Validator("ENV_FOR_DYNACONF", is_not_in=["unittest"]),
-        #           condition=utils.check_url_is_reachable),
-        Validator(SettingKeys.CSW_SERVER.value, must_exist=True, when=not_doc),
-        Validator(SettingKeys.DATA_ACCESS.value, must_exist=True, when=not_doc),
-        Validator(SettingKeys.GROUP_PROPERTY.value, must_exist=True, when=not_doc),
-        Validator(SettingKeys.WHITELIST.value, must_exist=True, when=not_doc),
-
-        # Validator("CSW_SERVER_DC", must_exist=True, when=Validator("ENV_FOR_DYNACONF", is_not_in=["unittest"]),
-        #           condition=utils.check_url_is_reachable),
-        Validator(SettingKeys.CSW_SERVER_DC.value, must_exist=True, when=not_doc),
-        Validator(SettingKeys.DATA_ACCESS_DC.value, must_exist=True, when=not_doc),
-        Validator(SettingKeys.GROUP_PROPERTY_DC.value, must_exist=True, when=not_doc),
-        Validator(SettingKeys.WHITELIST_DC.value, must_exist=True, when=not_doc),
+        Validator("IS_CSW_SERVER", default=False),
```

Comment:
For documentation reasons each configuration key is defined in the enum class `SettingKeys` 
The name and value of the enum are equal and beneath the key definition should be a doc-string describing the key
Then in the settings validation these keys are used e.g. `SettingKeys.IS_CSW_SERVER.value`

Reply:
Perfect, I'll use the SettingKeys then